packages/components/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 ### Bug Fix
 
+-   `ExternalLink`: Check if the link is an internal anchor link and prevent anchor links from being opened. ([#42259](https://github.com/WordPress/gutenberg/pull/42259)).
+
+### Bug Fix
```

Comment:
```suggestion

```

I'm not sure we need x2 `Bug Fix` headings do we?

Reply:
-It has 4 bug fix headings...

Reply:
Ok I stand corrected 😄 

Thanks for clarifying.

Reply:
I think you might be right because it looks weird haha.